### PR TITLE
Martz med ui

### PR DESCRIPTION
**Pull request by oehokie**

### CODE REVIEW

File: client/app/scripts/controllers/record.js [hunk 1]
```diff
@@ -116,16 +305,40 @@ angular.module('phrPrototypeApp').controller('RecordCtrl', function ($scope, $wi
...
 
-        medapi.fdaName(drugName, function (err, data) {
-            if (err) {
-                console.log("ERR: " + err);
-            } else {
-                $scope.openfdanameResults = data;
+    $scope.setSelectedDrug = function setSelectedDrug() {
+        if (this.rxdrug.selected) {
+            this.rxdrug.selected = false;
+            $scope.selDrug = {};
```
Comment (jsachs) [inline client/app/scripts/controllers/record.js]: Might want to just rename to selectedDrug, don't think the brevity is necessary.

File: client/app/views/modals/medicationentry.html [hunk 2]
```diff
@@ -6,18 +6,23 @@
...
-                                    <span><i class="fa fa-ellipsis-h"></i> Drug</span>
+                                <button class="btn btn-link" data-toggle="tab" data-target="#medtype" aria-controls="medtype" ng-click="swapMedTabs('medtype')">
+                                    <span>Medication Type</span>
                                 </button>
                             </li>
                             <li>
-                                <button class="btn btn-link" data-toggle="tab" data-target="#prescriber" aria-controls="prescriber" ng-click="swapMedTabs('prescriber')">
-                                    <span><i class="fa fa-comment"></i> Prescriber</span>
+                                <button class="btn btn-link" data-toggle="tab" data-target="#medinfo" aria-controls="medinfo" ng-click="swapMedTabs('medinfo')">
+                                    <span>Prescription</span>
```
Comment (jsachs) [inline client/app/views/modals/medicationentry.html]: I think it's confusing that the second tab is 'Prescription' regardless of which option you select on the first tab.

File: client/app/views/modals/medicationentry.html [hunk 3]
```diff
@@ -25,24 +30,172 @@
...
+                                                <td>{{rxdrug.rxcui}}</td>
+                                            </tr>
+                                        </tbody>
+                                    </table>
+                                    <!--<pre>{{rxnormResults}}</pre>-->
+                                </div>
+                            </div>
+                            <br />
+                            <div class="col-sm-12" ng-if="medSearchType=='prescription'">
+                                <h4>Prescriber Information</h4>
```
Comment (jsachs) [inline client/app/views/modals/medicationentry.html]: You get a 400 if there are no matches--we're going to need to handle this better, maybe with a UI error notification.

File: client/app/views/modals/medicationentry.html [hunk 4]
```diff
@@ -25,24 +30,172 @@
...
+                                <div class="row">
+                                    Drug Info: {{selDrug.synonym}}
+                                </div>
+                                <div class="row" ng-show="selPrescriber">
+                                    Prescriber: {{selPrescriber.first_name}} {{selPrescriber.last_name}}, {{selPrescriber.credential}}
+                                </div>
+                            </div>
+                            <h3>Additional Info</h3>
+                            <div class="row">
+                                <div class="col-sm-5">When did you start taking it?</div>
```
Comment (jsachs) [inline client/app/views/modals/medicationentry.html]: Can we make this a date selector like registration?

**Comment by oehokie** [comment]
This PR is in relation to #570 #569 #534 #533 #521 possibly others

File: client/app/views/modals/medicationentry.html (hunk 3 above)
Comment (oehokie) [inline client/app/views/modals/medicationentry.html]: We'll do this as a bug and I'll fix it today

File: client/app/views/modals/medicationentry.html (hunk 2 above)
Comment (oehokie) [inline client/app/views/modals/medicationentry.html]: Just pushed a change for this

File: client/app/views/modals/medicationentry.html (hunk 4 above)
Comment (oehokie) [inline client/app/views/modals/medicationentry.html]: I'll add that now

**Pull request merged**